index.html

```diff
@@ -14,17 +14,20 @@
 </head>
 
 <body>
-
     <header class="toolbar-dark-compact">
         <div class="logo-placeholder"><a href="/"><img src="assets/images/logo-placeholder.png" alt="Portal logo"></a>
         </div>
         <div class="normal-button">
-            <nav class="nav-right-hamburguer"><i class="fa-solid fa-bars"></i></nav>
-            <nav class="nav-right-text">
+            <nav class="nav-right-hamburguer"><a id="navbar-h" href="#"><i class="fa-solid fa-bars"></i></a></nav>
+
+            <nav id="navbar-options" class="nav-right-text">
```

Comment:
- [ ] [OPTIONAL] I kindly suggest that you add `position: sticky` to your navbar so that it always displays on the screen and makes it easier to access for the user.